TrainingTask/MyCar.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TrainingTask
+{
+    public class MyCar : Drawer
+    {
+        private TrackField trackField = new TrackField();
+
+        private readonly char carSymbol;
+
+        public static int MyCarPosition { get; set; }
+
+        public MyCar()
+        {
+            this.carSymbol = 'x';
+
+            GenerateElement(4, trackField.TrackHeight - 2, this.carSymbol);
+            GenerateElement(4, trackField.TrackHeight - 4, this.carSymbol);
+            GenerateElement(5, trackField.TrackHeight - 3, this.carSymbol);
+            GenerateElement(5, trackField.TrackHeight - 4, this.carSymbol);
+            GenerateElement(5, trackField.TrackHeight - 5, this.carSymbol);
+            GenerateElement(6, trackField.TrackHeight - 2, this.carSymbol);
+            GenerateElement(6, trackField.TrackHeight - 4, this.carSymbol);
+
+            MyCar.MyCarPosition = 1;
+        }
+
+        public void MoveLeft()
```

Comment:
Method does not move a car. It renders it in another position.
Single responsibility principle is not followed.

Reply:
-1 to SRP

Reply:
Should i just plus some positions? Like to move car on the right +5, to move on the left -5?